Library/Source/NativeEngine.h

```diff
@@ -384,6 +390,7 @@ namespace Babylon
         bx::DefaultAllocator m_allocator;
         uint64_t m_engineState;
 
+        inline static BGFXCallback m_bgfxCallback{};
```

Comment:
nit: Should static variables have the `m_` name styling?

Reply:
I didn't find any static member variable to copy the style. What about _ instead of m_? What do you propose?

Reply:
No idea; I don't think we have a convention as I don't think we're hoping to have static members long-term. @bghgary, any suggestions for this?

Reply:
Using _ for now.

Reply:
Typical Hungarian notation uses `s_` but I'm not entirely sure we are following Hungarian notation.